backend/uclapi/requirements.txt

```diff
@@ -6,7 +6,7 @@ billiard==3.5.0.3
 celery==4.1.0
 certifi==2017.11.5
 chardet==3.0.4
-ciso8601==1.0.5
```

Comment:
any reason for this downgrade here?

Reply:
Nah, probably just needs syncing with master